docs/cookbook/BiDailyTrigger.md

```diff
@@ -11,7 +11,7 @@ active_sub_menu_id: site_mnu_docs_cookbook
 
 # How-To: Trigger That Executes Every 2 Days
 
-At first glance, you may be tempted to use a CronTrigger. However, if this is truly to be every two days, CronTrigger won't work. To illustrate this, simply think of how many days are in a typical month (28-31). A cron expression like "0 0 5 2/2 * ?" would give us a trigger that would restart its count at the beginning of every month. This means that we would would get subsequent firings on July 30 and August 2, which is an interval of three days, not two.
+At first glance, you may be tempted to use a CronTrigger. However, if this is truly to be every two days, CronTrigger won't work. To illustrate this, simply think of how many days are in a typical month (28-31). A cron expression like "0 0 5 2/2 * ?" would give us a trigger that would restart its count at the beginning of every month. This means that we would get subsequent firings on July 30 and August 2, which is an interval of three days, not two.
```

Comment:
Duplicated words aren't always typos, but they often are, and I think all instances I saw in this repository were. Spell checkers have been performing this check for decades even though it isn't actually an instance of a word not in a dictionary.